README.md

```diff
@@ -162,9 +181,86 @@ specifies channel 0 which is connected to the thermometer.
     iex> volts = counts / 1023 * 3.3
     1.461290322580645
 
+Since many people coming to `elixir_ale` may be new to reading data sheets 
+and working with bitstrings, the following is a more detailed explanation.
+
+First we call SPI.start_link with Linux spidev device driver as it argument.  
+We pattern match with `{:ok, pid}` so we can use `pid` in the 
+SPI.transfer function next.
```

Comment:
Question:  Do you want to delete just this paragraph or the entire section discussing the `start_link` call?  I'm going to just delete this paragraph in my next update, but let me know if you mean the entire section.